shopelectro/selenium/analytics_goals.py

```diff
@@ -0,0 +1,47 @@
+import abc
+
+from shopelectro.selenium import SiteDriver
+
+# @todo #929:30m Implement GoogleEcommerceGoals and YandexMetrikaGoals and use them in tests.
+
+
+class Goals(abc.ABC):
+    """Front-end reached goalss."""
+
+    def __init__(self, driver: SiteDriver):
+        self.driver = driver
+
+    @abc.abstractmethod
+    def fetch(self):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def __iter__(self):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def __getitem__(self, index: int):
+        raise NotImplementedError
+
+
+class YandexEcommerceGoals(Goals):
+    """
+    Represent reached Yandex ecommerce goals.
```

Comment:
it's not informative string. Let's rm it. Leave an ignore comment if py doc lint requires it